src/basic/env-util.c

```diff
@@ -564,6 +564,29 @@ char* strv_env_pairs_get(char **l, const char *name) {
         return result;
 }
 
+int strv_env_get_merged(char **l, char ***ret) {
+        _cleanup_strv_free_ char **v = NULL;
+        size_t n = 0;
+        int r;
+
+        assert(ret);
+
```

Comment:
maybe add a breif comment here: `/* converts an strv with pairs of env var name + env var value into an strv that returns them concatenated with a "=" separator" */`